### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by iiiwan** [gxhen/LoRa_RFFI#4]
hi , I want to know the difference between Closed_set_RFFI and Openset_RFFI_TIFS

**Comment by refealwang** [comment]
Obviously, Closed_set_RFFI is a simple classification network based on resnet, while Openset_RFFI_TIFS uses resnet and Triplet Network to train the feature extractor, and then uses knn for classification, which can achieve the addition and deletion of signals, so it is called "Openset".
